Src/Support/Google.Apis/Requests/BatchRequest.cs

```diff
@@ -220,8 +219,17 @@ public async Task ExecuteAsync(CancellationToken cancellationToken)
                 }
                 else
                 {
-                    // Parse the error from the current response.
-                    var error = await service.DeserializeError(responseMessage).ConfigureAwait(false);
+                    RequestError error;
+                    try
+                    {
+                        // Parse the error from the current response.
+                        error = await service.DeserializeError(responseMessage).ConfigureAwait(false);
+                    }
+                    catch(GoogleApiException ex) when (!(ex.Error is null))
```

Comment:
Nit: add space. Oh for later versions of C# with "is not null". (Or just use `ex.Error is object` for now.)